src/handlers/HandlerFactory.ts

```diff
@@ -0,0 +1,6 @@
+import StakeRequestedHandler from './StakeRequestedHandler';
```

Comment:
As  TransactionHandler.ts takes `handlers` as Constructor args, not sure how do we plan to use  this class. Can you please elaborate how do we plan to use it.

Reply:
TransactionHandler needs different kinds of handlers may be 10 for now. It seems it's not the responsibility of transaction handler to instantiate different handler. I created this factory class which can instantiate different kinds of handlers and which is passed in the constructor.

Reply:
Agree this approach works well with TDD.